tools/vf-component-library/src/site/troubleshooting/index.njk

```diff
@@ -5,30 +5,35 @@ date: 2019-04-09 12:24:50
 tags:
   - troubleshooting
 layout: layouts/base.njk
-templateEngineOverride: njk, md
+templateEngineOverride: njk
 ---
 
-<br/><br/>
-{% render '@vf-intro', {"vf_intro_phase": "", "vf_intro_heading": title,
+{% render '@vf-intro', {
+  "component-type": "container",
+  "vf_intro_heading": title,
   "vf_intro_lede": subtitle,
   "vf_intro_text": [
-    "Many problems are due to local develpment environment configuration, here's how to get your machine back on track."
-  ]
+    "Many problems are due to local develpment environment configuration, here's how to get your development back on track.",
+    "Before filing a ticket or tossing your machine out the window, we've found most issues can be sorted by one of these."
+  ],
+  "vf_intro_badge": {
+    "style": [
+      "phases"
+    ],
+    "text": "beta",
+    "theme": "primary",
+    "badge_href": "JavaScript:Void(0);"
+  }
 } %}
 
-
 <section class="vf-content | embl-grid embl-grid--has-centered-content">
+{% render '@vf-section-header', {
```

Comment:
it would be nice to eventually get this using `.yml` instead of relying on `{% render … %}`